<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sistema_de_login</class>
 <widget class="QDialog" name="sistema_de_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro De Usuário</string>
  </property>
  <widget class="QLineEdit" name="usuario_lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Novo Usuário</string>
   </property>
  </widget>
  <widget class="QLabel" name="permissoes_de_acesso_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Permissões de Acesso</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cadastrar_pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>711</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>8</number>
   </property>
   <widget class="QWidget" name="Vendas">
    <attribute name="title">
     <string>Vendas</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>175</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="vendas_vende_checkBox">
        <property name="text">
         <string>Vendas &gt;&gt; Vender</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Compras">
    <attribute name="title">
     <string>Compras</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>222</width>
       <height>158</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="compras_orcamentos_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Orçamentos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_pedidos_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Pedidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_fornecedores_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Fornecedores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_devolucoes_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Devoluções</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_centraldecompras_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Central de Compras</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_estoque_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Estoque</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Cadastro">
    <attribute name="title">
     <string>Cadastro</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>175</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cadastrar_cliente_checkBox">
        <property name="text">
         <string>Cadastro &gt;&gt; Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cadastrar_funcionario_checkBox">
        <property name="text">
         <string>Cadastro &gt;&gt; Funcionário</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cadastrar_fornecedor_checkBox">
        <property name="text">
         <string>Cadastro &gt;&gt; Fornecedor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Cliente">
    <attribute name="title">
     <string>Cliente</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>175</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="localizar_cliente_checkBox">
        <property name="text">
         <string>Cliente &gt;&gt; Localizar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="editar_cliente_checkBox">
        <property name="text">
         <string>Cliente &gt;&gt; Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Estoque">
    <attribute name="title">
     <string>Estoque</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Financeiro</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Relatorio">
    <attribute name="title">
     <string>Relatório</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Modulo">
    <attribute name="title">
     <string>Módulo</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Ferramentas">
    <attribute name="title">
     <string>Ferramentas</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>234</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QCheckBox" name="cadastrar_usuario_checkBox">
        <property name="text">
         <string>Ferramentas &gt;&gt; Cadastrar Usuário</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="administrador_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Se selecionado, garante acesso total ao sistema. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Administrador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Acesso Total</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="senha_lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuário</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Senha</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
